codechecker_common/checker_labels.py

```diff
@@ -0,0 +1,261 @@
+import os
+from collections import defaultdict
+from itertools import chain
+from typing import Dict, Iterable, List, Tuple, Union
+from codechecker_common.util import load_json_or_empty
+
+
+class CheckerLabels:
+    # These labels should be unique by each analyzer. If this uniqueness is
```

Comment:
I didn't mean a verb here, because not the analyzers make them unique. I really wanted to write that the labels should be unique. Maybe the continuation is bad: "should be unique per analyzer"?